docs/mnist_example.md

```diff
@@ -1,254 +0,0 @@
-# MNIST IR Example
-
-This shows the MNIST MLP model as it is compiled from Keras, lowered to XLA HLO,
-and then lowered to an IREE module with SPIR-V. Several steps are omitted for
-brevity.
```

Comment:
Yes. https://github.com/google/iree/issues/1586 is tracking that. This file is quite outdated; it can cause more confusion than help if still left in the repo.